037-WindowsVirtualDesktop/Coach/10-Monitor-Manage-Performance-Health.md

```diff
@@ -0,0 +1,141 @@
+
+    ![User Input Delay by Process](../Images/Challenge10-InputDelaybyProcess.png)
+
+1. Enable automatic scaling of WVD session hosts to start and stop machines during peak/off-peak hours across each region. 
+    - Students should be able to demonstrate a successful deployment of the [WVD Scaling Script](https://docs.microsoft.com/en-us/azure/virtual-desktop/set-up-scaling-script) **NOTE:** which they may have already completed in Challenge 9*
+    - Have students open the Logic Apps Designer View to show the HTTP Post step and values that have been configured as per the above setup notes. Confirm they have values as expected in the Body fields and consider reasoning behind why values may be different (ie MinimumNumberofRDSH/SessionThresholdPerCPU) for purpose of the challenge. 
+    - Extra points if the students have configured 'depth-first' load balancing (non-default setup so students will have to change this themselves). Depth-first will distribute users to an available session host with the highest number of connections but has not reached its maximum session limit threshold. This would meet the goal of the company to reduce underutilised machines. It can also reduce session sprawl towards the end of the day but isn't always guaranteed to do so. 
+    - Ideally you should have the team demonstrate a successful peak/off-peak transition if time permits but this may require adjustments to the times set and frequency of the job within the Logic App Designer before attempting. Viewing the scaling tool logs will give an indication of success/failure of the job running if time is running out. 
+
+    ![Logic App Designer](https://docs.microsoft.com/en-us/azure/virtual-desktop/media/logic-apps-designer.png)
+
+## Advanced Challenges (Optional)
+
+1. Modify the WVD Insights portal to display the CPU % performance metric for the current host pool on the front page of the dashboard. You will need to ensure the metric is only displayed on the Overview tab. 
+    - Students should be able to show a custom CPU performance metric being added to their edited WVD Insights workbook. 
+    - Open the WVD Insights page then "Select Customise" 
+    - In the next page "Select Edit" 
+    - For faster changes scroll to the bottom and look for the +Add menu  and select "add metric"
+
+    ![Add Metric in Insights Workbook Edit mode](../Images/Challenge10-AddMetricDropdown.png)
+
+    - Metric Scope: Resource Scope
+    - Virtual Machine dropdown: "WVDHostsId" selected and add the VM CPU Performance counter. 
+    ![Add CPU Metric](../Images/Challenge10-AddMetricValue.png)
+
+    - Save work by first hitting the done editing button and choosing the "Save As" option before hitting Done edit. Users, if successful will have a new metric counter that they added.
+
+## Reference
```

Comment:
Change this to 'Learning Resources', doesn't matter that it's in the coach guide.